app/assets/javascripts/discourse/app/components/quote-button.js

```diff
@@ -361,113 +356,69 @@ export default Component.extend(KeyEnterEscape, {
     );
   },
 
-  _saveFastEditDisabled: propertyEqual(
-    "_fastEditInitialSelection",
-    "_fastEditNewSelection"
-  ),
-
   @action
   insertQuote() {
     this.attrs.selectText().then(() => this._hideButton());
   },
 
   @action
-  _toggleFastEditForm() {
+  async _toggleFastEditForm() {
```

Comment:
Does this need to be async? I don't see any `await` inside 👀

Reply:
both `async` and `return` are needed 😃 the code below this block is also in this action 😛

Reply:
🤦